src/components/Tabs/Tabs.st.css

```diff
@@ -1,10 +1,16 @@
 :import {
-    -st-from: "../../tpaStyles/styleableExt";
+    -st-from: "../../tpaStyles/styleableExt.js";
     -st-named: wixApply;
 }
 
+:import {
+    -st-from: "./ScrollableTabs/ScrollableTabs.st.css";
+    -st-default: Navigation;
+}
+
 :vars {
     tabsHeight: 52px;
+    tabsMobileHeight: 48px;
```

Comment:
Theses are hardcoded default values? shouldn't be provided through santa?

Reply:
No, these aren't overridable